package.json

```diff
@@ -32,8 +33,9 @@
     "karma-coverage": "^1.0.0",
     "karma-jasmine": "^0.3.6",
     "karma-phantomjs-launcher": "^0.2.1",
+    "moment": "^2.17.1",
     "phantomjs": "^1.9.18",
-    "protractor": "^3.1.1",
+    "protractor": "^4.0.8",
```

Comment:
I had to update Protractor version in order to run the e2e locally. Is it a known issue for you folks?